src/modules/packages/main.py

```diff
@@ -54,7 +54,9 @@ def install(self, pkgs):
             check_chroot_call(["apt-get", "-q", "-y", "install"] + pkgs)
         elif self.backend == "pacman":
             check_chroot_call(["pacman", "-Sy", "--noconfirm"] + pkgs)
-
+# https://wiki.sabayon.org/index.php?title=En:List_Of_Equo_Functions
+                  elif self.backend == "Entropy":
```

Comment:
Please name the backend "entropy" all lowercase, and add it to the bad backend check near the end of this file.